services/hooks/src/listeners.js

```diff
@@ -57,15 +57,19 @@ class HookListeners {
     );
     debug('Listening to hook exchanges');
     this.pulseHookChangedListener = consumer;
-    this.listeners = [];
+    this.listeners = {};
     // Reconcile on start up
     await this.reconcileConsumers();
   }
 
   /** Create a new pulse consumer for a hook */
-  async createListener(hook, queueName) {
+  async createListener(hookGroupId, hookId, queueName) {
     debug(`${queueName}: creating listener (and queue if necessary)`);
```

Comment:
Should this come after the new check below?

Reply:
Probably!